src/utils.js

```diff
@@ -180,6 +183,9 @@ export function evalStorybookStorySnapshots({ waitFor }) {
   };
 
   return waitFor(async () => {
+    // uncache stories, if cached via storyStorev7: true
```

Comment:
does this work with `storyStorev7: false` ?

Reply:
Lets add a test for the same.

Reply:
Will check this.
Also need to verify this with older versions of storybook

Reply:
Refer - https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#v7-store-api-changes-for-addon-authors

Reply:
v6 and v7 with `true` tests are covered, can you add tests for v7 with `false`

Reply:
This will make the tests even more complicated.
We _might_ have to run all test cases again.

Why is this an issue, as this has been officially documented by storybook team, that we need to call `cacheAllCSFFiles` as documented above.

Reply:
we don't want a case where calling when its `false` and it errors out.

Though I'll keep this requirement optional if this is tested locally by creating a percy build.